### ISSUE TITLE
Onnx Conversion for Faster-RCNN

### ISSUE THREAD

**Issue by ashnair1** [open-mmlab/mmdetection#2411]
Has `pytorch2onnx.py` ever been used to convert a standard Faster-RCNN model? While I  understand it's still an experimental feature, I thought I could use `pytorch2onnx` to convert my Faster-RCNN model but it did not work.

**Comment by ashnair1** [comment]
To provide more context, these are the things I've tried:

I tried running `pytorch2onnx.py` however this creates an invalid graph and fails the onnx check (`onnx.checker.check_model(onnx_model)`) due to two Constant nodes containing the device name ('cpu'). The same error as in #2299. As stated in #2299, I was able to create a valid graph by removing these two nodes.

However, I was not able to load this onnx model. I tried using `onnxruntime` and caffe2 onnx backend.

1. Loading with onnxruntime fails with `Fail: [ONNXRuntimeError] : 1 : FAIL : Fatal error: ATen is not a registered function/op`
2. Loading with Caffe2 Onnx backend (opset 9) fails with `Don't know how to translate op roi_align` which I guess makes sense since RoiAlign support was added opset 10 onwards.

I had also tried #1386 which did allow me to create a onnx model and run inference. However, the model was giving the same outputs for different inputs.
